Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -1437,7 +1439,10 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
             return;
         }
     }
-    @catch (NSException *exception) {
+    @catch (id exception) {
+        // In case we catch exception here, hide the overlaying message.
+        [self dismiss];
```

Comment:
So, the assumption here is the rich interstitial was not getting hidden post display? due to exception?

Reply:
Yes that is correct.